portality/bll/doaj.py

```diff
@@ -130,4 +130,9 @@ def tourService(cls):
     @classmethod
     def autochecksService(cls, autocheck_plugins=None):
         from portality.bll.services import autochecks
-        return autochecks.AutocheckService(autocheck_plugins=autocheck_plugins)
\ No newline at end of file
+        return autochecks.AutocheckService(autocheck_plugins=autocheck_plugins)
+
+    @classmethod
+    def urlshortService(cls):
+        from portality.bll.services import urlshort
+        return urlshort
```

Comment:
This needs to follow the pattern of the other services, for code style consistency